### PR TITLE
Create triggers in DBM

### PR DESCRIPTION
**Pull request by a-mon**
Implements all trigger endpoints except for bulk get active triggers.

Adds a test shell script for the DBM service as well.

Fixes some bugs in the Postgres shell scripts.

### CODE REVIEW

File: DatabaseService/PostgreSQL/docker_context/create_tables.sql [hunk 1]
```diff
@@ -1,13 +1,13 @@
 rollback;
 
 create table if not exists users (
-    user_id varchar(255) primary key
+    user_id varchar(12) primary key
```
Comment (MitchellGRead) [inline DatabaseService/PostgreSQL/docker_context/create_tables.sql]: why have we cut the size of this down? there could definitely be cases of longer then 12 character user id's

File: DatabaseService/docker_context/handlers/PostgresHandler.py [hunk 2]
```diff
@@ -134,6 +141,114 @@ async def handleGetUserStocksCommand(self, user_id, stock_id):
...
+        get_triggers_query = from_table_where_user_query.format(
+            columns='*',
+            table=sell_triggers_table,
+            user=user_id
+        )
+
+        result = await self.fetchQuery(get_triggers_query)
+
+        if result == []:
+            return {'errorMessage':'Triggers not found.'}, 404
```
Comment (MitchellGRead) [inline DatabaseService/docker_context/handlers/PostgresHandler.py:240]: would this necessarily be an error or just that the user does not have any triggers set

**Review by MitchellGRead — COMMENTED** [review]
overall think it looks good, just couple comments

File: DatabaseService/PostgreSQL/docker_context/create_tables.sql (hunk 1 above)
Comment (a-mon) [reply DatabaseService/PostgreSQL/docker_context/create_tables.sql]: I saw in one of the schemas that we were limiting this to 12 at most hence the decision to change this. Is that not the case? I can change it back to 255 though or some other number that you think would be better.

File: DatabaseService/docker_context/handlers/PostgresHandler.py (hunk 2 above)
Comment (a-mon) [reply DatabaseService/docker_context/handlers/PostgresHandler.py:240]: Good point actually. This also means that I should change other user-specific GET endpoints to not throw an error in this case, as they currently are.

File: DatabaseService/PostgreSQL/docker_context/create_tables.sql (hunk 1 above)
Comment (MitchellGRead) [reply DatabaseService/PostgreSQL/docker_context/create_tables.sql]: Ya I think Mitch may have done that once but was a similar concern and got reverted. Sorry if it was missed. If possible I think we put it back to 255

Comment (a-mon) [reply DatabaseService/PostgreSQL/docker_context/create_tables.sql]: Reverted to 255. Resolving comment.

File: DatabaseService/docker_context/handlers/PostgresHandler.py (hunk 2 above)
Comment (a-mon) [reply DatabaseService/docker_context/handlers/PostgresHandler.py:240]: Will handle in follow up PR.

**Review by MitchellGRead — APPROVED** [review]
looks good

**Pull request merged**